.travis.yml

```diff
@@ -6,10 +6,34 @@ php:
   - 5.5
   - 5.6
   - hhvm
+  - 7.0
 
 sudo: false
 
-install: composer install
+matrix:
+  fast_finish: true
+  include:
+    - php: '5.3'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '5.4'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '5.5'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '5.6'
+      env: SYMFONY_VERSION=2.3.*
+    - php: 'hhvm'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '7.0'
+      env: SYMFONY_VERSION=2.3.*
```

Comment:
Done

* https://travis-ci.org/FriendsOfSymfony/FOSUserBundle/builds/144110094
* on php 5.3/5.4 bundle tested on **2.3** and **~2.3 | ~3.0** (i.e. 2.8)
* on php 5.5/5.6/hhvm/7.0 bundle tested on **2.3**, **2.8**, **3.0** and **~2.3 | ~3.0** (i.e. 3.1)